App/View Controllers/Posts/PostsPageViewController.swift

```diff
@@ -1658,6 +1676,10 @@ extension PostsPageViewController: RenderViewDelegate {
             view.embedTweets()
         }
         
+        if UserDefaults.standard.enableFrogAndGhost {
+            view.loadLottiePlayer()
```

Comment:
This seems odd to me. If we're including `enableFrogAndGhost` in the template context, why do we need to also call this here?

Reply:
Lottie animations are json files and it's the ghost.json file that's being placed into the template context and into a hidden div. (because I couldn't get awful-resource:// to work in render.js)

This loadLottiePlayer() loads the actual player library from a cdn. I just copied what we do for Twitter for this one

https://cdnjs.cloudflare.com/ajax/libs/lottie-web/5.7.11/lottie.min.js